src/main/java/com/graylog2/input/cloudtrail/CloudTrailTransport.java

```diff
@@ -87,7 +88,8 @@ public void run() {
         LOG.info("Starting cloud trail subscriber");
 
         subscriber = new CloudTrailSubscriber(
-                Region.getRegion(Regions.fromName(input.getConfiguration().getString(CK_AWS_REGION))),
```

Comment:
What would be the migration path for existing users here?

Maybe populate `aws_s3_region` and `aws_sqs_region` with the old value of `aws_region` if they don't exist?